examples/multisig-example/build.gradle.kts

```diff
@@ -7,8 +7,8 @@ val bitcoinjVersion = "0.15.6"
 val kotlinxVersion = "1.2.0"
 
 dependencies {
-    implementation(project(":crypto"))
-    implementation(project(":protobuf"))
+    implementation("network.finschia:finschia-kt-crypto:0.2.2")
+    implementation("network.finschia:finschia-proto:1.0.1")
     implementation(project(":tx"))
```

Comment:
How about change order of each dependencies for lint? 
tx is multi module, it seems better to be separated with others.

Reply:
Thanks for the feedback. Do you have any idea or reference for this?

Reply:
Okay, I'll re-organize dependencies with the category of scope(runtime, api, implementation, test and etc) and features if applicable.